colossalai/inference/tensor_parallel/modeling/llama.py

```diff
@@ -363,7 +353,7 @@ def llama_flash_attn_kvcache_forward(
                     infer_state.block_loc,
                     infer_state.start_loc,
                     infer_state.seq_len,
-                    infer_state.cache_manager.past_key_values_length,
+                    infer_state.max_len_in_batch,
                 )
             else:
                 Llama2TokenAttentionForwards.token_attn(
```

Comment:
Does this can import from lightllm directly?

Reply:
I don't  understand you

Reply:
I have checked this function in kernel/triton/token_attention_kernel.py. The function has been already imported from ligthllm directly.